docs/.vitepress/config.mts

```diff
@@ -1,5 +1,12 @@
 import { defineConfig, type DefaultTheme } from 'vitepress'
 
+function getStoryHost(): string {
+  if (process.env.CONTEXT !== 'production' && process.env.DEPLOY_PRIME_URL) {
+    return new URL(process.env.DEPLOY_PRIME_URL).host.replace('-docs', '-story')
+  }
+  return 'story.sefirot.globalbrains.com'
+}
+
```

Comment:
This uses Netlify's build-time environment variables to get corresponding story URL - i.e. if you now open docs in deploy preview and click on "View in Histoire" button it will open the story deploy preview. The reverse is also same - "View Documentation" button should point to docs' deploy preview if you're viewing deploy preview of story.

Reply:
👍